tvloader/parser2v1/parse_creation_info.go

```diff
@@ -102,7 +102,7 @@ func (parser *tvParser2_1) parsePairFromCreationInfo2_1(tag string, value string
 
 // ===== Helper functions =====
 
-func extractExternalDocumentReference(value string) (string, string, string, string, error) {
+func extractExternalDocumentReference(value string) (spdx.DocElementID, string, string, string, error) {
```

Comment:
This might be worth keeping as string, given the comments earlier above.

Reply:
Resolving this comment to keep discussion about string vs DocElementID constrained to one comment